Delete_Exchange_Webshells.py

```diff
@@ -0,0 +1,116 @@
+"""
```

Comment:
This does not have any specific actions for Hafnium or webshells so we should give it a more generic name like "delete_suspicious_files".

Reply:
name has been changed